control/control.go

```diff
@@ -38,16 +49,6 @@ import (
 	"github.com/moby/buildkit/util/tracing/transform"
 	"github.com/moby/buildkit/version"
 	"github.com/moby/buildkit/worker"
-	digest "github.com/opencontainers/go-digest"
-	"github.com/pkg/errors"
-	"go.etcd.io/bbolt"
-	sdktrace "go.opentelemetry.io/otel/sdk/trace"
-	tracev1 "go.opentelemetry.io/proto/otlp/collector/trace/v1"
-	"golang.org/x/sync/errgroup"
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/codes"
-	"google.golang.org/grpc/metadata"
-	"google.golang.org/grpc/status"
```

Comment:
oops I'll fix

Reply:
yeah, reverting this will make future upstream merges easier.